### PR TITLE
Add two new fields for Common Voice endpoint

### LINKED ISSUE #164: Additional Common Voice fields (2 day streak & Last Active date)
There are 2 new fields that we need ported over from Common Voice into SFDC. The fields are: 

2 day streak / boolean (cv_two_day_streak__c)
Common Voice Last Active Date (date) (cv_last_active_dt__c)

Please work with Desiree and Gregor w/ the Common Voice team to get these data values tested in the ADVO SFDC Sandbox, then moved to Production.

### PR DESCRIPTION
**Pull request by pmac**
Fix #164

### CODE REVIEW

**Comment by pmac** [comment]
@Gregoor these are the new fields. Sending booleans to the `two_day_streak` field should be the string `True` or `False`. The `last_active_date` field is the same date format as the others.

**Comment by Gregoor** [comment]
Excellent, sounds good 👌 Thanks!

File: basket/news/forms.py [hunk 1]
```diff
@@ -78,6 +78,10 @@ class CommonVoiceForm(forms.Form):
     first_contribution_date = forms.CharField(required=False,
                                               max_length=20,
                                               validators=[validate_datetime_str])
+    last_active_date = forms.CharField(required=False,
+                                       max_length=20,
+                                       validators=[validate_datetime_str])
+    two_day_streak = forms.NullBooleanField()
```
Comment (jpetto) [inline basket/news/forms.py]: Do you think it's worth testing this value as `Null`/an empty string?

Comment (pmac) [reply basket/news/forms.py]: The test that doesn't include that value is kindof a test for that. It's empty, which means that field will be `None` which is then filtered out and not sent to the task at all. I think that should cover pretty well. Could possibly be better, but this view is protected by API key so we can better expect good values from our partner on this one as well.

**Comment by pmac** [comment]
Thanks Jon!

**Pull request merged**